tests/containers/helm.pm

```diff
@@ -0,0 +1,75 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test deploy a helm chart
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use utils qw(zypper_call script_retry);
+use version_utils qw(is_sle);
+use registration qw(add_suseconnect_product get_addon_fullname);
+
+sub run {
+    my ($self) = @_;
+
+    $self->select_serial_terminal;
+    $self->{deployment_name} = "apache";
+    my $chart = "bitnami/apache";
+
+    zypper_call("in -y jq aws-cli helm podman kubernetes-client");
+
+    # Check helm version
+    assert_script_run("helm version");
```

Comment:
I guess this line is redundant as it's executed below.

Reply:
done